x-pack/plugins/index_management/public/lib/render_badges.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React, { Fragment } from 'react';
+import { EuiBadge } from '@elastic/eui';
+import { getBadgeExtensions } from '../index_management_extensions';
+export const renderBadges = (index) => {
```

Comment:
two badges look pretty good! it would be nice to maybe have them be different colors so the user can easily differentiate the meaning at a glance

![image](https://user-images.githubusercontent.com/1965714/51280922-cbb57880-1995-11e9-938c-a273d24c41d5.png)

![image](https://user-images.githubusercontent.com/1965714/51280988-facbea00-1995-11e9-983b-b8af11f7e651.png)